### ISSUE TITLE
Bot does nothing

### ISSUE THREAD

**Issue by AC-DC989** [norecha/pokubot#77]
It doesn't zoom out or anything so I removed the config file and it did not let me locate the barracks. My resolution is correct, I ran pokubot 1.0.3, reinstalled bluestacks but this one does not want to work.

**Comment by GMetaxakis** [comment]
you need the latest version, after the latest clash of clans update.

**Comment by AC-DC989** [comment]
Haha I know that, everything is updated all the way, including bluestacks.

**Comment by ashishmverma1** [comment]
Updating BS is not a good idea since the bot heavily relies on image processing, and different versions of BS might render graphics differently.

Anyways that must not be an issue here. The new version of pokubot (v1.1.1) doesn't needs the barracks coordinates, so that is not an issue.
As for the bot to interact with the BS window, you need to keep whole of the BS window above other windows (and not minimized of course). You sure you've got all of this right?

**Comment by AC-DC989** [comment]
Yep it is the only thing open (besides the bot) and there have been 3 different versions of bluestacks tested. Version 1.1.1 does nothing but if i use the outdated 1.0.3 then it will at least zoom out on all versions. But it won't work obviously because it's outdated.

**Comment by norecha** [comment]
Do you still have this?
